examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCCommissionerDeclaration.h

```diff
@@ -0,0 +1,70 @@
+/**
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#import <Foundation/Foundation.h>
+
+#ifndef MCCommissionerDeclaration_h
+#define MCCommissionerDeclaration_h
+
+/**
+ * Represents the Commissioner Declaration message sent by a User Directed Commissioning server
+ * (CastingPlayer/Commissioner) to a UDC client (Casting Client/Commissionee).
+ */
+@interface MCCommissionerDeclaration : NSObject
+
+/** The allowed values for the ErrorCode field are the following */
+typedef NS_ENUM(NSInteger, CdError) {
+    kNoError = 0,
+    kCommissionableDiscoveryFailed = 1,
```

Comment:
Is it possible to add the string representation of these CdErrors to this enum instead of needing the separate stringForErrorCode method?

Reply:
I Googled around, Objective-C doesn't support adding metadata directly to enum values.